src/modules/deal-guarantee/deal-guarantee.service.test.ts

```diff
@@ -38,7 +40,7 @@ describe('DealGuaranteeService', () => {
     currentDateProviderGetLatestDateFromTodayAnd = jest.fn();
     currentDateProvider.getLatestDateFromTodayAnd = currentDateProviderGetLatestDateFromTodayAnd;
 
-    service = new DealGuaranteeService(acbsAuthenticationService, acbsDealGuaranteeService, currentDateProvider);
+    service = new DealGuaranteeService(acbsAuthenticationService, acbsDealGuaranteeService, currentDateProvider, transformations);
```

Comment:
In other service this transformations are called dateStringTransformations. Here name quite generic.